sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs

```diff
@@ -3,9 +3,10 @@
 // license information.
 
 using System;
+using System.ComponentModel;
 using System.Text;
 
-namespace Azure.Storage.Blobs
+namespace Azure.Storage.Sas
```

Comment:
Should this go into a specific `Azure.Storage.Sas` folder?

Reply:
I've already moved all the Sas functionality into project specific `Sas` folders.  Does that work?

Reply:
I guess? It is hard to keep track of those changes while looking at the PR.